plugins/action/action.h

```diff
@@ -187,6 +207,26 @@ class Action
     void return_to_launch_async(result_callback_t callback);
 
     /**
+     * @brief Send command to transition the drone to fixedwing (asynchronous).
+     *
+     * Note that this is only for the vtol type.
```

Comment:
Argg, my comment from yesterday lost. Can you check your email for my suggested wording?

Reply:
Found it :) 
Next commit will include your comment.Found it :) 
Next commit will include your comment.

Reply:
Thanks. Looks good. We will need to test that the comment is actually true once the timeout is fixed :-)